apps/api/v2/src/modules/stripe/controllers/stripe.controller.ts

```diff
@@ -79,24 +81,67 @@ export class StripeController {
   @UseGuards()
   @Redirect(undefined, 301)
   @ApiOperation({ summary: "Save stripe credentials" })
+  /**
```

Comment:
I think it is good to add the "why" behind proxying in this case - we proxy because we need to check whether or not the user can install stripe on the team and we do that using route guards instead of having to copy route guard logic into a service.